websites/H/HentaiVN/presence.ts

```diff
@@ -0,0 +1,499 @@
+					matched => keynum[matched]
+				);
+			presenceData.state = `Từ khóa: ${namekey} - Doujin: ${doukey} - Nhân vật: ${charkey} - Thể loại: ${keystr}`;
+		} else presenceData.state = "Đang nhập dữ liệu tìm kiếm...";
+	} else if (document.location.pathname.startsWith("/forum/nhan_tin.php")) {
+		presenceData.details = "Đang nhắn tin...";
+		presenceData.state = document.querySelector("h3").textContent.trim();
+		presenceData.smallImageKey = "chater";
+	} else if (document.location.pathname.startsWith("/forum/mail.php")) {
+		if (document.location.search.includes("page"))
+			page = new URLSearchParams(document.location.search).get("page");
+
+		presenceData.details = "Đang xem hộp thư...";
+		presenceData.state = `Trang ${page}`;
+		presenceData.smallImageKey = "chat";
+	} else if (document.location.pathname.startsWith("/forum/free-market.php")) {
+		presenceData.details = "Đang lướt chợ tự do...";
+		presenceData.smallImageKey = "shop";
+	} else if (document.location.pathname.startsWith("/forum/shop.php")) {
+		presenceData.details = "Đang lướt shop...";
+		presenceData.smallImageKey = "shop";
+	} else if (document.location.pathname.startsWith("/forum/user_item.php")) {
+		presenceData.details = "Đang xem kho đồ cá nhân...";
+		presenceData.smallImageKey = "kho";
+	} else if (
+		document.location.pathname.startsWith("/forum/search_member.php")
+	) {
+		presenceData.details = "Đang tìm kiếm thành viên...";
+		presenceData.smallImageKey = "search";
+	} else if (document.location.pathname.startsWith("/forum/member.php")) {
+		presenceData.details = "Đang xem danh sách thành viên...";
+		presenceData.smallImageKey = "surf";
+	} else if (document.location.pathname.startsWith("/admin/admin.php")) {
+		presenceData.details = "Trong trang quản trị truyện...";
+		presenceData.smallImageKey = "up";
+	} else if (document.location.pathname.startsWith("/bookmark-list.php")) {
+		if (document.location.search.includes("page"))
+			page = new URLSearchParams(document.location.search).get("page");
+
+		presenceData.details = "Đang xem danh sách truyện đang theo dõi";
+		presenceData.state = `Trang ${page}`;
+		presenceData.smallImageKey = "mark";
+	} else if (document.location.pathname.startsWith("/forum/huyhieu.php")) {
+		presenceData.details = "Đang xem phòng trưng bày huy hiệu";
+		presenceData.smallImageKey = "huyhieu";
+	} else if (document.location.pathname.startsWith("/forum/my_waifu.php")) {
+		presenceData.details = "Đang xem danh sách waifu cá nhân";
+		presenceData.smallImageKey = "waifu";
+	} else if (document.location.pathname.startsWith("/forum/home_waifu.php")) {
+		presenceData.details = "Đang lướt shop nhà waifu...";
```

Comment:
```suggestion
		presenceData.details = "Đang duyệt shop nhà waifu...";
```
Better word choice & consistency with other strings

Reply:
That is the term used in the website, which cannot be changed.